Sources/TensorFlow/Layers/Core.swift

```diff
@@ -223,3 +223,26 @@ public extension Dense {
                   activation: activation)
     }
 }
+
+
+public struct Function<InputScalar: TensorFlowFloatingPoint, OutputScalar:TensorFlowFloatingPoint>: Layer {
+    /// The Function Layer arbitrarily takes any differentiable function and returns a Layer which can
+    /// used as a wrapper around that function.
+    public typealias Input = Tensor<InputScalar>
+
+    public typealias Output = Tensor<OutputScalar>
+
+    public typealias Body = @differentiable (Input) -> Output
+
+    @noDerivative public let body: Body
+
+    public init(
```

Comment:
Make initializer label-less and merge line from below.

```suggestion
    public init(_ body: @escaping Body) {
```

A Precedent for `init(_ body:)` is [`AnyIterator.init`](https://developer.apple.com/documentation/swift/anyiterator/1784409-init).